docs/adguard-browser-extension/mv3-version.md

```diff
@@ -0,0 +1,1102 @@
+
+- *Tracking protection* (formerly known as *Stealth mode*)
+
+  ![Tracking protection *border](https://cdn.adtidy.org/content/blog/new/tracking_screen.png)
+
+  There are no *Cookies* section, along with *Self-destruction of first-party cookies* and *Self-destruction of third-party cookies* since we cannot set the TTL of cookies using declarative rules.
+
+- *Phishing & malware protection* is no longer available in the general settings. To protect yourself from malicious websites and scams, enable the appropriate filters in the *Security* tab.
+
+  ![Security](https://cdn.adtidy.org/content/blog/new/security.png)
+
+## Limitations
+
+### Maximum number of rules and filters
+
+Manifest V3 imposes limits on static and regex rules per extension. Users may receive notifications when these limits are exceeded.
+
+![Rule limits *border](https://cdn.adtidy.org/content/blog/new/rulelimits.png)
+
+![Too many extensions *border](https://cdn.adtidy.org/content/blog/new/other_extension.png)
+
+Basically, Manifest V3 divides rules into static (built-in) and dynamic, with strict limits in place.
+
+**Static rules:** **30,000** rules per extension, with a cumulative limit of **330,000** for all extensions installed by a single user.
+
+For regex rules, the limit is set at **1,000 per extension**.
+
+The maximum number of simultaneously enabled filters is **50**.
+
+**Dynamic rules:** a strict cap of **5,000** rules is imposed, which includes a maximum of 1,000 regex rules.
+
+  If this limit is exceeded, only **5,000 converted rules** will be applied in the following order: first user rules, then allowlist, and finally — custom filters.
+
+> **Converted rules** are rules that have been transformed to [DNR](https://developer.chrome.com/docs/extensions/reference/api/declarativeNetRequest#build-rules) format using the [declarative converter](https://github.com/AdguardTeam/tsurlfilter/tree/master/packages/tsurlfilter/src/rules/declarative-converter). During this conversion process, some rules may overwrite others (`badfilter`), some may be combined (`removeparame`), resulting in a list of rules with a slightly different order.
+>
+> From this list of converted rules, we will only use 5,000 rules. The rest of them will be displayed in the editor, but not applied.
+
+### Network rules
+
+Network rules have limitations: some types of rules cannot be implemented in MV3, or can only be implemented with restrictions.
+
+Below is a list of modifiers that are supported, partially supported, or not supported at all. More details about the limitations are described on [GitHub](https://github.com/AdguardTeam/tsurlfilter/tree/master/packages/tsurlfilter/src/rules/declarative-converter#readme).
+
+#### Supported basic modifiers
```

Comment:
here we should update all links to the repo, when these changed will be merged to master, because now it has link to branch.

Reply:
also, i don't think that we should describe here all rules, because these examples are auto-generated and can be changed in every moment.
maybe we can insert here some basic examples (from here - https://github.com/AdguardTeam/tsurlfilter/tree/release/v3.1/packages/tsurlfilter/src/rules/declarative-converter#basic-examples, for example) just for clarification how DNR rules looks like? and for further information place a link to auto-generated docs with examples?